app/src/lib/shells/win32.ts

```diff
@@ -12,6 +13,8 @@ export enum Shell {
   PowerShellCore = 'PowerShell Core',
   Hyper = 'Hyper',
   GitBash = 'Git Bash',
+  Wsl = 'WSL',
```

Comment:
At this point I think most of my questions are about what we display to the user:

 - It'd be nice to know which distro or shell you're going to end up in. Is that possible without launching `wsl.exe`?
 - If `wsl.exe` potentially puts you in the Bash shell for whichever distro you have, does that negate the need to also have a `WSL Bash` entry?

Reply:
- As I commented on the original issue, `wslconfig /list` gives a list of distributions and the default one is launched when using `wsl.exe`. So that can be parsed to work out which distribution will be launched https://github.com/desktop/desktop/issues/4641#issuecomment-387864749
- Yes I suppose it does. It would also require launching WSL to work out if you have setup a different shell and since it is probably a rare use case might be best to avoid confusion